samples/ios/Continote/Continote/MyNotesTableViewDataSource.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright (c) 2017 Google Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import FirebaseDatabase
+import FirebaseDatabaseUI
+import MaterialComponents.MaterialSnackbar
+
+/**
+ This is the data source used for the MyNotesTableView. We need to override FUITableViewDataSource
+ in order to allow the user to edit (and then subsequently delete) rows (i.e. Notes) within the
+ table.
+ */
+class MyNotesTableViewDataSource: FUITableViewDataSource {
+
+  override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+    // Allow the user to edit any row in the table.
+    return true
+  }
+
+  override func tableView(_ tableView: UITableView,
+                          commit editingStyle: UITableViewCellEditingStyle,
+                          forRowAt indexPath: IndexPath) {
+    switch editingStyle {
+    case .delete:
```

Comment:
Since we only care about a single case here, you should be able to just check with: `if editingStyle == .delete {`